thirdparty/CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ ExternalProject_Add(poco
         )
 
 ExternalProject_Add(fds
-        GIT_REPOSITORY https://github.com/XiaoMi/galaxy-fds-sdk-cpp.git
+        GIT_REPOSITORY https://gitee.com/mirrors_XiaoMi/galaxy-fds-sdk-cpp
```

Comment:
seem allow multi url, you can test add both them: "url1,url2"

Reply:
can't work with it